### DISCUSSION TITLE
Will the `pyenv` project come to the apt or pacman

### DISCUSSION THREAD

**Discussion by osbm** [pyenv/pyenv#2303] [Q&A] [answered]
This project is really great. I am using it on daily basis. But installing is hard. Would you guys planning to distribute this project via flatpak, snap, apt or pacman?

**Comment by native-api** [comment, marked as answer]
We're going to rather simplify the installation in https://github.com/pyenv/pyenv/issues/2085.

Pyenv's setup is identical to other language version managers (rvm/rbenv, phpenv, nvm).
